corehq/apps/domain/decorators.py

```diff
@@ -95,52 +95,53 @@ def call_view(): return view_func(req, domain_name, *args, **kwargs)
             msg = _('The domain "{domain}" was not found.').format(domain=domain_name)
             raise Http404(msg)
 
-        if user.is_authenticated and user.is_active:
-            if not domain_obj.is_active:
-                msg = _(
-                    'The project space "{domain}" has not yet been activated. '
-                    'Please report an issue if you think this is a mistake.'
-                ).format(domain=domain_name)
-                messages.info(req, msg)
-                return HttpResponseRedirect(reverse("domain_select"))
-            couch_user = _ensure_request_couch_user(req)
-            if couch_user.is_member_of(domain_obj):
-                # If the two factor toggle is on, require it for all users.
-                if (
-                    _two_factor_required(view_func, domain_obj, couch_user)
-                    and not getattr(req, 'bypass_two_factor', False)
-                    and not user.is_verified()
-                ):
-                    return TemplateResponse(
-                        request=req,
-                        template='two_factor/core/otp_required.html',
-                        status=403,
-                    )
-                else:
-                    return call_view()
-
-            elif (
-                _page_is_whitelist(req.path, domain_name) or
-                not domain_obj.restrict_superusers
-            ) and user.is_superuser:
-                # superusers can circumvent domain permissions.
+        if not (user.is_authenticated and user.is_active):
```

Comment:
I recommend this one with [whitespace off](https://github.com/dimagi/commcare-hq/pull/26292/commits/51bdcb8d470cfeecf66def98aaff75b3021b5d69?w=1)